CMakeLists.txt

```
@@ -88,7 +88,7 @@ set(CURSES_NEED_NCURSES TRUE)
 # Dependencies
 find_package(FS REQUIRED)
 find_package(Numa)
-find_package(LLVM 6.0.0 CONFIG)
+find_package(LLVM 7.1 CONFIG)
```

```suggestion
find_package(LLVM 7 CONFIG)
```

This change came with commit 239b230c5ea297ed6e6094f5e8d3f2361e3640f0
I am surprised that we want to use clang and llvm in version `7.1` instead of `7`.
I can neither install version `7.1` with apt on Ubuntu 19.04 nor with brew on MacOS. I would therefore suggest to stick with version `7`.
Jenkins [fails](https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/hyrise%2Fhyrise/detail/PR-1614/11/pipeline) because of this issue.

---

That comes from the LLVM repo. Will revert once we distupgraded our machines